src/menu.ts

```diff
@@ -25,6 +25,7 @@ function onOpen() {
         .addItem('Image generation', 'ImageGenerationService.manuallyRun')
         .addItem('Image upload', 'ImageUploadService.manuallyRun')
         .addItem('Image assets linking', 'ImageExtensionService.manuallyRun')
+        .addItem('Image Pause', 'ImagePauseService.manuallyRun')
```

Comment:
nit: lowercase pause to be consistent
```suggestion
        .addItem('Image pause', 'ImagePauseService.manuallyRun')
```